tests/dummy/config/fastboot-testing.js

```diff
@@ -1,11 +1,23 @@
 const najax = require('najax');
+const semver = require('semver');
 
-module.exports = {
-  resilient: false,
-  buildSandboxGlobals(defaultGlobals) {
-    return Object.assign({}, defaultGlobals, {
-      SampleGlobal: `TestSampleGlobal`,
-      najax
-    });
-  },
-};
+const version = require(require.resolve('fastboot/package.json')).version;
+
+if (semver.lt(version, '3.0.0')) {
```

Comment:
found out one of the tests related to `sandboxGlobals` was not working for the `fastboot-1.2` scenario